packages/seatmaps-client/package.json

```diff
@@ -18,9 +18,15 @@
   "repository": "github:ticketevolution/seatmaps-client",
   "author": "Ticket Evolution, Inc.",
   "license": "MIT",
-  "main": "./cjs/index.js",
-  "module": "./esm/index.mjs",
   "types": "./cjs/index.d.ts",
+  "main": "./dist/bundle.umd.cjs",
+  "module": "./dist/bundle.js",
+  "exports": {
+    ".": {
+      "import": "./dist/bundle.js",
+      "require": "./dist/bundle.umd.cjs"
+    }
+  },
```

Comment:
Pretty sure this block is also wrong right now for the same reasons as above comments.
But I don't have any specific picture in mind how exports block should be structured.
Previously I find more success with just using main an module fields, even though exports is a never version of those.

One thing you might want to add is a jsdlver field for pointing cdn to new bundle.